integration/samples/api/specs/sourcemaps.ts

```diff
@@ -3,17 +3,18 @@ import * as fs from 'fs-extra';
 import * as path from 'path';
 
 describe(`@sample/api`, () => {
-  describe('angular.component.d.ts.map', () => {
+  describe('sample-api.d.ts.map', () => {
     let sourceMap;
     beforeAll(() => {
-      sourceMap = fs.readJsonSync(path.resolve(__dirname, '../dist/src/angular.component.d.ts.map'));
+      sourceMap = fs.readJsonSync(path.resolve(__dirname, '../dist/types/sample-api.d.ts.map'));
     });
 
     it(`should exist`, () => {
       expect(sourceMap).to.be.ok;
     });
 
-    it('should point to the correct source path', () => {
+    // TODO: figure a way to make DTS maps work with bundling, or disable bundling when sourcemap is enabled.
```

Comment:
I thought you got it working?

Reply:
Yeah, but it does need an upstream change.